src/components/tools/Charts/Shared/Annotation/Labels.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+  import { onMount } from "svelte";
+  import { parseCssValue } from "./annotationUtils.js";
+
+  export let annotations;
+  export let xGet;
+  export let yGet;
+
+  const vals = ["top", "right", "bottom", "left"];
+
+  $: labels = annotations.map((d) => d.label);
+
+  $: setPosition = (label) => {
+    const { position, data } = label;
+    let style = "";
+
+    if (position) {
+      vals.forEach((val) => {
+        if (position[val]) {
+          style += `${val}: ${position[val]};`;
+        }
+      });
+    }
+
+    if (data) {
+      style = `top: ${yGet(data)}px;left: ${xGet(data)}px;`;
+    }
+
+    return style;
+  };
+</script>
+
+<style>
+  .annotation-label {
+    position: absolute;
+    max-width: var(--max-width, 200px);
```

Comment:
where are the CSS variables being set? I don't see how they're meant to be overridden.

Reply:
@clhenrick I'll add in the reference to the Layercake code I'm using. The default props can be overriden by passing in a `styles` object with the css props like `style: { "stroke-dasharray": "6,4", "fill":"red" }`. Just realized this works with the `Connector` component, need to add it to the `Labels` component

Reply:
@mukhtyar forgot to mention that the font-size of 1rem feels a little too large when compared to the rest of the chart's labels; bumping it down to 0.875rem or 0.75rem might help it not clash so much with the axis tick labels for example.